addons/openebs/template/base/install.sh

```diff
@@ -48,13 +48,51 @@ function openebs() {
 function openebs_maybe_migrate_from_rook() {
     if [ -z "$ROOK_VERSION" ]; then
         if kubectl get ns | grep -q rook-ceph; then
-            rook_ceph_to_sc_migration "$OPENEBS_LOCALPV_STORAGE_CLASS"
+            # show validation errors from pvmigrate
+            # if there are errors, openebs_maybe_rook_migration_checks() will bail
+            openebs_maybe_rook_migration_checks
+            rook_ceph_to_sc_migration "$OPENEBS_LOCALPV_STORAGE_CLASS" "1"
             DID_MIGRATE_ROOK_PVCS=1 # used to automatically delete rook-ceph if object store data was also migrated
         fi
     fi
 }
 
-function prompt_migrate_from_rook() {
+function openebs_maybe_rook_migration_checks() {
+
+    # get the list of StorageClasses that use rook-ceph
+    rook_scs=$(kubectl get storageclass | grep rook | grep -v '(default)' | awk '{ print $1}') # any non-default rook StorageClasses
+    rook_default_sc=$(kubectl get storageclass | grep rook | grep '(default)' | awk '{ print $1}') # any default rook StorageClasses
+
+    # Ensure openebs-localpv-provisioner deployment is ready
+    echo "awaiting openebs-localpv-provisioner deployment"
```

Comment:
Not a blocker 
But could we not use as we have in kURL logStep logWarn logFail as well?

Reply:
We could. However, mapping messages to different colors is a lot more involved than I expected. I rather we have a separate PR for making the output more CLI UX friendly.